### ISSUE TITLE
When watching files - new errors and warnings don't show when output hash remains the same

### ISSUE THREAD

**Issue by tedbarth** [webpack-contrib/eslint-webpack-plugin#5]
- Operating System: Linux (most probably unrelated)
- Node Version: 12.14.1 (most probably unrelated)
- NPM Version: 6.13.4 (most probably unrelated)
- webpack Version: 4.41.6
- eslint-webpack-plugin Version: 0.1.0

### Expected Behavior

ESLint validation (errors/no errors any more) should print out in webpack watch mode regardless of whether final build did run or not.

### Actual Behavior

ESLint validation (errors/no errors any more) does not print out current ESLint validation state when webpack does not update chunks in watch mode. This is the case when using babel and changing only trailing spaces. The transpiled code then did not change and no chunks will be rebuild. While this is fine, the ESLint validation doesn't update either: You see webpack processing (suppose you have --progress flag set) but newly added ESLint errors are print out or "visually removed" from output when fixed.

For eslint-loader this [bug](https://github.com/webpack/webpack/issues/2538) is already fixed. We use eslint-webpack-plugin only to have ESLint checks ONLY when changing files in watch mode (`lintDirtyModulesOnly`).

### Code

```js
// webpack.config.js
new ESLintPlugin({
  extensions: ['.js', '.mjs'], // Necessary to have linting at all;
  lintDirtyModulesOnly: true, // Lint only changed files in watch mode (not on normal build or initial watch build)

  // Always print errors and warnings (don't use auto config)
  emitError: true,
  emitWarning: true
})
```

**Comment by tedbarth** [comment]
For the moment we use [babel-plugin-add-filehash](https://www.npmjs.com/package/babel-plugin-add-filehash) as a workaround.
This might add some side effects, though.

**Comment by ricardogobbosouza** [comment]
Hi @tedbarth
Please provide a repository with the issue.
